### ISSUE TITLE
URL bar not closing automatically on scroll in mobile browsers when using Lenis

### ISSUE THREAD

**Issue by norouzy** [darkroomengineering/lenis#306]
When integrating Lenis into my web application, I noticed that the URL bar in the mobile browser does not close automatically when scrolling down or up. Instead, it remains open all the time.

The URL bar should close automatically when scrolling down and open when scrolling up, providing a cleaner browsing experience for mobile users.

**Comment by clementroche** [comment]
This is currently impossible

**Comment by norouzy** [comment]
> you mean using `syncTouch:true` ? This is currently impossible

yes i use `syncTouch:true` but i mean When integrating Lenis into my web application, I noticed that the address bar in the mobile browser remains open all the time, even when scrolling down. This detracts from the user experience as it takes up valuable screen space. Ideally, I'd like the address bar to automatically hide when scrolling down and reappear when scrolling up, providing a cleaner browsing experience for mobile users. Is there any functionality or configuration within Lenis that can enable this behavior, similar to what some modern mobile browsers implement?

**Comment by clementroche** [comment]
yes as I mentioned, this is currently not possible due to browsers limitations, only native events can trigger this actions ( the one with `isTrusted:true` ).

**Comment by norouzy** [comment]
ok thanks

> yes as I mentioned, this is currently not possible due to browsers limitations, only native events can trigger these actions ( the ones with `isTrusted:true` ). It's for security reasons I guess. That's the same reason why you cannot hide and show this bar programmaticaly.
